src/stores/modules/Groups.store.js

```diff
@@ -31,8 +31,8 @@ const actions = {
 
   async ADD_GROUP({commit, state}, groupname) {
     commit('fetching');
-    var result = await api.groups.addNewGroup(groupname);
-    if(result.errors) {
+    const {success} = await api.groups.addNewGroup(groupname);
```

Comment:
This does change the semantics to depend on error code as opposed to the JSON body, but that's probably more correct anyway.